mysql-test/mysql-test-run.pl

```diff
@@ -3719,8 +3719,11 @@ ($)
     # to be able to distinguish them from manually created
     # version-controlled results, and to ignore them in bzr.
     my $dest = "$base_file$suites.result~";
-    my @cmd = ($exe_patch, qw/--binary -r - -f -s -o/,
-               $dest, $base_result, $resfile);
+    my @cmd = ($exe_patch);
+    if ($^O ne "freebsd") {
+      push @cmd, '--binary';
```

Comment:
Is `--binary` needed on Linux?

Reply:
is it here for Windows?

Reply:
not needed for windows or Linux - purged per #1500. I assume you don't remember why it was added in c398770. Happy EOL 5.5